### ISSUE TITLE
Add 'how to organize HTML code for better SEO' section

### ISSUE THREAD

**Issue by yaman3bd** [mdo/code-guide#263]
in the next new update, if it is possible to add  new table of contents or a section that explains how to structure and organize the HTML code for  better SEO results it would be really helpful
like what should the HTML code contains 
how to order it
what should be avoided 
what is the best practice
what is recommended to use
how to use `role` or `aria-*` atts
etc....

**Comment by mdo** [comment]
I feel like SEO is a bit outside the scope of this guide, but there might be some value to roles and ARIA attributes.

**Comment by yaman3bd** [comment]
I do not think so, because the guide explains how to organize `CSS` and `HTML` code  so also if a new section is added explaining how to organize the code for better SEO it would still relevant,
and I think adding something for SEO would make the guide the best content online explaining such things because it is written by someone who has many years of experience working with things like that.
